src/app/app.module.ts

```diff
@@ -40,6 +41,7 @@ export function createTranslateLoader(http: HttpClient) {
     StatusBar,
     SplashScreen,
     { provide: RouteReuseStrategy, useClass: IonicRouteStrategy },
+    InformedConsentService,
```

Comment:
`InformedConsentService` should not be imported here, that is what the `SharedServiceModule` is for.